src/root/edit-jobset.tt

```diff
@@ -92,10 +92,10 @@
   </div>
 
   <div class="form-group row">
-    <label class="col-sm-3" for="editjobsettype">Type</label>
+    <label class="col-form-label col-sm-3" for="editjobsettype">Type</label>
```

Comment:
Not sure if the `col-form-label` is required, but the others had it it, so added it here too.